src/ansys/dpf/composites/_composite_model_impl.py

```diff
@@ -291,6 +208,33 @@ def material_names(self) -> dict[str, int]:
 
         return names
 
+    @property
+    def ply_types(self) -> dict[str, int]:
```

Comment:
For me it would be more natural to have a mapping of dpf_material_ids to ply types. I would expect that the user gets the material_id from the element_info or from the name and then uses it to filter by ply type. Isn't there a problem if two material ids have the same ply type? If I understand correctly only the second material id would be assigned.